bridge/test/merkleProof/merkle-proof-library.test.ts

```diff
@@ -0,0 +1,523 @@
+import { expect } from "chai";
+import { MockMerkleProofLibrary } from "../../typechain-types";
+import { deployLibrary } from "./setup";
+
+const ZERO_BYTES32 =
+  "0x0000000000000000000000000000000000000000000000000000000000000000";
+
+describe("Testing Merkle Proof Library", async () => {
+  let MerkleProofLibrary: MockMerkleProofLibrary;
+
+  beforeEach(async function () {
+    MerkleProofLibrary = await deployLibrary();
+  });
+
+  describe("verifyNonInclusion:", async () => {
+    it("does not revert with non inclusion default hash MerkleProof", async () => {
+      const auditPath =
+        "0xe602c66f5176c6d2a33d6eb3addf38c937e0e32457e58148883578cb4655b826";
+      const root =
+        "0x529312e0c69f0cc47d27630461d884d9537ebfa51d57c1ecf7f38f77c801373f";
+      const key =
+        "0x80ab269d23d84721a53f9f3accb024a1947bcf5e4910a152f38d55d7d644c995"; // utxoID
+      const bitset = "0x80";
+      const height = 0x1;
+      const included = false;
+      const proofKey = ZERO_BYTES32;
+      const proofValue = ZERO_BYTES32;
+
+      await expect(
+        MerkleProofLibrary.verifyNonInclusion(
+          {
+            included: included,
+            keyHeight: height,
+            key: key,
+            proofKey: proofKey,
+            proofValue: proofValue,
+            bitmap: bitset,
+            auditPath: auditPath,
+          },
+          root
+        )
+      ).not.to.be.reverted;
+    });
+
+    it("does not revert with non inclusion leaf node MerkleProof", async () => {
+      const auditPath =
+        "0x066c7a6ef776fbae26f10eabcc5f0eb72b0f527c4cad8c4037940a28c2fe3159" +
```

Comment:
A lot of these tests could be more readable by defining these consts globally with more context in the name, i.e. `validKey` vs. `invalidKey` instead of including in each particular test. Right now it's very hard for a reader to identify what exactly is differing between each test.

Reply:
Good suggestion, the original tests seemed to follow a more isolated "per test" approach to the inputs which i mirrored here but there is definitely some duplication that could be addressed with constants for at least the valid information; I'll do some cleanup

Reply:
Thanks! Much easier to read.

Reply:
Thanks! Much easier to read.